### DISCUSSION TITLE
What are those Pinokio versions in cocktailpeanutlabs/P2 ?

### DISCUSSION THREAD

**Discussion by hansolocambo** [6Morpheus6/pinokio-wiki#4] [Q&A]
Hi, 
I've been using the Pinokio Trellis script for a few days: https://github.com/pinokiofactory/TRELLIS
It works great as I generated tons of 3D objects... until it stopped working. Without me changing anything. 

I used to install Pinokio (3.0.32) from this source: https://github.com/pinokiocomputer/pinokio

But I read somewhere that Trellis needed some beta updates of pinokio (3.0.37) available from this source: https://github.com/cocktailpeanutlabs/p2/releases

Do you know about this _p2_ github ? Is it the same version of Pinokio with new code being tested before it's release on the more official github page ? Or is it another unrelated fork or version of Pinokio ?

Please advise. Thanks ;)

**Comment by 6Morpheus6** [comment]
@hansolocambo
Cocktailpeanut, the dev of Pinokio releases there his beta versions. They aren't officially published. Some might be experimental. But they often include new functions and bugfixes.
Right now I  can recommend to install the latest one from p2.
Currently it's this one. [https://github.com/cocktailpeanutlabs/p2/releases/tag/3.2.218](https://github.com/cocktailpeanutlabs/p2/releases/tag/3.2.218)
I use it by myself. It's stable and contains the latest features.
About trellis. The devs of Trellis work on that project continiosly which sadly tends to break the installer from time to time and it needs to be fixed and adjusted. But none of the apps you install can just stop working without any reason. As long as you use an app and you don't app date it, there's no reason why it should suddenly stop working.
I can't tell if Trellis is working again, but the latest beta definitely supports all function needed.
